Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -106,6 +106,14 @@ private static bool IsHttpUri(Uri uri)
                 if (IsCacheEnabled && _isHttpSource)
                 {
                     _image.Source = await ImageCache.GetFromCacheAsync(_uri);
+                    if (_image.Source != null)
```

Comment:
If we leave it like this, I think if would be a breaking change from the previous version where those events were signaled also when image was loaded from the cache. 
Adding OnImagedLoadedFromCache  event would probably be misleading since image still can be downloaded from the web it's just that cache does it.
@andrejt , @ScottIsAFool what do you think?